JitsiConference.js

```diff
@@ -142,10 +148,11 @@ function JitsiConference(options) {
 
     /**
      * Flag set to <tt>true</tt> when P2P session has been established
-     * (ICE has been connected).
+     * (ICE has been connected) and this conference is currently in the peer to
+     * peer mode (P2P connection is the active one).
      * @type {boolean}
      */
-    this.p2pEstablished = false;
+    this.p2p = false;
```

Comment:
i'm a little confused that this represents both whether the p2p connection is established (from an ice perspective) AND that the conference is in p2p mode.  there will be periods of time where that is not necessarily true, right? (when transitioning to/from p2p, there will be a period of time where the p2p ice connection is made, but we're not currently using it?)